doc/modules/ROOT/pages/getting-started.adoc

```diff
@@ -91,6 +91,28 @@ Or we could change the database we are targeting later:
 gds.set_database("my-db")
 ----
 
+=== Configure Arrow parameters
+
+If Arrow is available on the server, we can provide the `GraphDataScience` constructor with several keyword parameters to configure the connection:
```

Comment:
Do we want to add a link/reference to the Arrow docs on the server here? If people don't know what this is about

Reply:
I think it's a good idea. We should also probably refer to it as "Apache Arrow" as we do in the server docs.